tests/seed/userSeed.js

```diff
@@ -0,0 +1,95 @@
+const { ObjectID } = require('mongodb');
+const { rolesList } = require('./roleSeed');
+const { authCompany } = require('./companySeed');
+const { customerList } = require('./customerSeed');
+const uuidv4 = require('uuid/v4');
+const {
+  VENDOR_ADMIN,
+  CLIENT_ADMIN,
+  AUXILIARY,
+  HELPER,
+  COACH,
+  PLANNING_REFERENT,
+  AUXILIARY_WITHOUT_COMPANY,
+  TRAINING_ORGANISATION_MANAGER,
+  TRAINER,
+} = require('../../src/helpers/constants');
+
+const userList = [
+  {
+    _id: new ObjectID(),
+    identity: { firstname: 'client_admin', lastname: 'Chef' },
+    refreshToken: uuidv4(),
+    local: { email: 'admin@alenvi.io', password: '123456' },
```

Comment:
client-admin@alenvi.io
pour etre clair :) - a changer cote front aussi dans les tests et dans le descriptif `it('should...`